### ISSUE TITLE
Mod ID: 'sodium', Requested by: 'flywheel', Expected range: '[0.6.0-beta.2,)', Actual version: '0.5.3'

### ISSUE THREAD

**Issue by astromaster21** [Engine-Room/Flywheel#259]
### Describe the Bug

i got this bug when trying to boot up a modpack i made 3 months ago and it gave me this error message, but im on forge so i dont know why it even asks for sodium, also in the installed mods list i dont see "flywheel" anywhere and the minecraft version of the modpack is set to 1.20.1 which the mod "flywheel" doesnt exist for that version

### Reproduction Steps

1.
2.
3.
...
i dont know

### Expected Result

i still dont know

### Screenshots and Videos

![Καταγραφή](https://github.com/user-attachments/assets/f2a6e9ac-960f-4e3f-9f10-a6ef2953baec)

### Operating System

Windows 10

### What is your GPU?

NVIDIA GeForce GTX 1060 3GB

### Mod Version

0.6.8.a

### Minecraft Version

1.19.2

### Loader Version

47.3.0

**Comment by IThundxr** [comment]
use embeddium instead of rubidium

**Comment by MaKuZan** [comment]
+

**Comment by Kiloname** [comment]
I'm getting the same issues and need help over here

**Comment by Kiloname** [comment]
and what mod is this belong too

**Comment by Kiloname** [comment]
> use embeddium instead of rubidium

how do we dop that @IThundxr

**Comment by SantiiMari** [comment]
> > use embeddium instead of rubidium
> 
> how do we dop that @IThundxr

Install [https://www.curseforge.com/minecraft/mc-mods/embeddium](url) & ensure you disable/remove Rubidium if you have that installed as it's a duplicate mod

**Comment by CarbixJr** [comment]
i get this error when i launch with sodium but without sodium the game won't launch at all

**Comment by Koldyr33** [comment]
> use embeddium instead of rubidium

If you remove Rubidium and install Embedium, an error appears with oculus

**Comment by notsmiel** [comment]
exactly the same error, all sodium ports are not working, flywheel is still requires 0.6.0 :c 
i cant live without sodium aghh

**Comment by imtheslimeguy** [comment]
yeah, first jellysquid said (the developer of sodium) that theres no 0.6.0 or above for 1.20.1: The problem is other mods specifying a dependency on non-existent versions of Sodium. We cannot fix this, and there is no version of Sodium 0.6.0+ for Minecraft 1.20.1., so its not our problem

**Comment by IThundxr** [comment]
replace rubidium with embeddium

**Comment by shiningnova233** [comment]
In fact, the Flywheel mod for version 1.19.2 has been merged into the Create mod. The specific path is: `/META-INF/jarjar/flywheel-forge-XXX.jar` inside the `create-XXX.jar`.If you're comfortable with technical tasks, you could try using a Java decompiler to modify the code and bypass the Sodium check (or perhaps find a pre-modified version online?).If that seems too troublesome, switching to Embedium is definitely the easiest solution.

**Comment by Z4nyDruid** [comment]
did  you solve it？
